packages/api/src/routes/medical/patient-root.ts

```diff
@@ -92,6 +95,52 @@ router.post(
       settings,
     });
 
+    if (!Config.isProdEnv()) {
+      const facilityNames = [
```

Comment:
move this whole block to its own file and have the conditional on the createPatient command, please

Reply:
addressed